test_gmail_ratelimiter.py

```
@@ -0,0 +1,23 @@
+import asyncio
+import time
+from collections import deque
```

Flake8 (7.3.0)

[error] 'collections.deque' imported but unused

(F401)